middleware/logger.go

```diff
@@ -21,9 +21,12 @@ import (
 	"net/http"
 )
 
+type logFunc func(format string, v ...interface{})
+
 // Logger middleware that will log http requests
 type Logger struct {
 	Verbose bool
+	logFunc logFunc
```

Comment:
nice ! but I am missing the usage ?

Reply:
I am using it for two reasons:
1) an interface to easily test this code
2) internally you can set a different logging function and not having side effect.